src/Service/Modules/TariffModule.cs

```diff
@@ -2,7 +2,7 @@
 {
     using Linn.Products.Facade.Services;
     using Linn.Products.Resources;
-
+    using Models;
```

Comment:
Linn.Products.Service.Models ?

Reply:
My Resharper keeps wanting me to take those bits out (Qualifier is redundant) such as when I use Resharper to pull a reference in it sometimes does this.  i tried to take it out the Resharper options but it didn't alter it so i'll just have to manually change these when they come up